ocs_ci/ocs/ocp.py

```diff
@@ -189,7 +191,24 @@ def get(
             command += f" --selector={selector}"
         if out_yaml_format:
             command += " -o yaml"
-        return self.exec_oc_cmd(command)
+        retry += 1
```

Comment:
I don't think this will be desired.
If you pass retry = 0, it will do retry once which is not what you wanted

Reply:
We need to run the code below once. If retry is set to 0 we will not not entry to while loop. The behavior is really same. Retry=0 will execute command once, of retry is 1 will execute once and will retry once more.